vllm/engine/arg_utils.py

```diff
@@ -685,7 +644,7 @@ def create_engine_config(self, ) -> EngineConfig:
             disable_sliding_window=self.disable_sliding_window,
             skip_tokenizer_init=self.skip_tokenizer_init,
             served_model_name=self.served_model_name,
-            multimodal_config=vision_language_config)
+            multimodal_config=mm_config)
```

Comment:
Perhaps it's just me, but how about using `multimodal_config` throughout this PR instead of `mm_config`? IMO the former is more friendly & intuitive to new developers from the community.

Reply:
yeah I had the same thought. Changed.